README.md

```diff
@@ -1 +1,2 @@
-# redi_frontend_a2020
\ No newline at end of file
+# redi_frontend_a2020
+> Separate repository for general practice in the class and regular homeworks of ReDi Frontend course (Autumn 2020)
```

Comment:
`>` thing means quote. Is it really what you aimed to? :)

Reply:
https://gist.github.com/fvcproductions/1bfc2d4aecb01a834b46
I took it here=) I thougth it is this nice looking vertical line=)) It's shown to me in that way, when I use the github online viewer=)

Reply:
Here is a nice reference: https://guides.github.com/features/mastering-markdown/#intro
Yeah if you want that => sure go for it! But in general if you will work with shared md files you will need to stick with regular markdown usage :)